WordPress/Classes/Models/WPAccount+RestApi.swift

```diff
@@ -12,7 +12,7 @@ extension WPAccount {
     @objc var wordPressComRestApi: WordPressComRestApi? {
         get {
             guard let api = objc_getAssociatedObject(self, &apiAssociatedKey) as? WordPressComRestApi else {
-                guard authToken.isEmpty else {
+                if let authToken, !authToken.isEmpty {
```

Comment:
I'm also changing these lines here https://github.com/wordpress-mobile/WordPress-iOS/pull/24327 👀

Reply:
I just saw your PR. I can resolve the conflicts once yours is merged. 😄

Reply:
My PR is missing the nullability check, so this change will be needed.